release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -527,6 +527,11 @@ With this release, {product-title} introduces the Data Processing Unit (DPU) Ope
 
 Administrators can now use the `ClusterUserDefinedNetwork` custom resource to deploy secondary networks on a `Localnet` topology. This feature allows pods and virtual machines connected to the localnet network to egress to the physical network. For more information, see xref:../networking/multiple_networks/primary_networks/about-user-defined-networks.adoc#nw-cudn-localnet[Creating a ClusterUserDefinedNetwork CR for a Localnet topology].
 
+[id="ocp-4-19-port-isolation-linux-bridge_{context}"]
+==== Enable port isolation for a Linux bridge NAD
+
+You can enable port isolation for a Linux bridge network attachment definition (NAD) so that virtual machines (VMs) or pods that run on the same virtual LAN (VLAN) can operate in isolation from one another. For more information, see xref:../virt/vm_networking/virt-connecting-vm-to-linux-bridge.adoc#virt-linux-bridge-nad-port-isolation_virt-connecting-vm-to-linux-bridge[Enabling port isolation for a Linux bridge NAD].
```

Comment:
Please verify that this link is correct. In the preview, the link takes to the [top of the page](https://94263--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-linux-bridge) rather than the [port-isolation part](https://94263--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/vm_networking/virt-connecting-vm-to-linux-bridge#virt-linux-bridge-nad-port-isolation.adoc_virt-connecting-vm-to-linux-bridge).